TACC/Core-CMS-Resources
# chore: clean up templates for TACC/Core-CMS#710

## Overview

Clean up templates in a way that supports https://github.com/TACC/Core-CMS-Custom and https://github.com/TACC/tup-ui overwrites of them.

## Related

- requires https://github.com/TACC/Core-CMS/pull/710
- similar to https://github.com/TACC/Core-CMS-Custom/pull/209

## Changes

- removed unnecessary overwrite of assets_font just to load super 
- ~~**removed** now-duplicate render of page title~~
	<sup>~~because [TACC/Core-CMS#710](TACC/Core-CMS/pull/710) does that in `base.html`~~</sup>
    <sup>this has since already been done (realized while merging `main` at 5a38ebf)</sup>
- ~~other changes likely made moot since merges of `main`~~
	

## Testing

> [!IMPORTANT]
> **Skipped**, because changeset has become negligible.

1. Deploy [all sites](https://github.com/TACC/Core-CMS-Resources#project-websites).
2. Verify favicon matches production.
3. Verify page titles match production.

## UI

N/A
